pkg/reconciler/sample/controller.go

```diff
@@ -51,6 +53,8 @@ func NewController(
 	r := &Reconciler{
 		dr:  &reconciler.DeploymentReconciler{KubeClientSet: kubeclient.Get(ctx)},
 		sbr: &reconciler.SinkBindingReconciler{EventingClientSet: eventingclient.Get(ctx)},
+		// Config accessor takes care of tracing/config/logging config propagation to the receive adapter
+		configAccessor: reconcilersource.WatchConfigurations(ctx, "sample-source", cmw),
```

Comment:
@slinkydeveloper just for my own understanding, what specifically does the 'component' parameter actually point to?

Reply:
it's a name used for bootstrapping the metrics exporter, it represents the name of the source